src/Sylius/Behat/Resources/config/services/contexts/setup.xml

```diff
@@ -146,6 +146,13 @@
             <tag name="sylius.behat.context" />
         </service>
 
+        <service id="sylius.behat.context.setup.static_content" class="Sylius\Behat\Context\Setup\StaticContentContext" scope="scenario">
```

Comment:
Shouldn't we consider put it in another file? For example split setup to ORM and ODM setup? I think its not a big deal, but still can be helpful if we would like to give an option to toggle phpcr.